content/common/cli-reference.md

```diff
@@ -1068,7 +1100,51 @@ TRIGGER OPTIONS:
 
 {{ blockEnd }}
 
-> **Please note:** If the `-f` or `--from` options do not accurately infer the intended registry or `.toml` file for your application, then you can explicitly specify either the `--from-registry` or  `--from-file` options to clarify this.
+> **Please note:** If the `-f` or `--from` options do not accurately infer the intended registry or `.toml` file for your application, then you can explicitly specify either the `--from-registry` or `--from-file` options to clarify this.
+
+### Watch
```

Comment:
This seems to split the "Trigger Options" section from "Up" and the readout of the trigger options help; in fact it makes "Trigger Options" a subheading of "Watch".

Reply:
Oops good catch